pkg/gator/test/test.go

```diff
@@ -148,9 +167,10 @@ func Test(objs []*unstructured.Unstructured, includeTrace bool) (*GatorResponses
 				trace = trace + "\n\n" + *r.Trace
 				targetResponse.Trace = &trace
 			}
-
 			responses.ByTarget[targetName] = targetResponse
 		}
+
+		responses.StatsEntries = append(responses.StatsEntries, review.StatsEntries...)
```

Comment:
Is this plus calling AggregateStats() duplicative?

Reply:
I don't think so since the `AggregateStats` on line 143 happens for children of the expansion into the `review` and now we take all the stats for all `review`s and make a list out of them 🤔 

Maybe there should be a test to make sure we don't have this scenario.